book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
Great job so far. As requested by the previous reviewer please add a many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. I saw you added in the `Person` class but it should also added in the `book` class since a `book` has many rentals. kindly add it.